data/net/minecraft/core/Rotations.mapping

```diff
@@ -5,6 +5,14 @@ CLASS net/minecraft/core/Rotations
 		COMMENT Rotation on the Y axis
 	FIELD z F
 		COMMENT Rotation on the Z axis
+	METHOD <init> (FFF)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD <init> (Lnet/minecraft/nbt/ListTag;)V
+		ARG 1 tag
```

Comment:
I think this should be a bit more verbose, since this specifically needs to be a `ListTag`.

Reply:
Seconded by Me and ChampionAsh